src/ja/drecomics/src/eu/kanade/tachiyomi/extension/ja/drecomics/DreComics.kt

```diff
@@ -0,0 +1,124 @@
+    "ja",
+) {
+    private val dateFormat = SimpleDateFormat("yyyy年MM月dd日", Locale.JAPAN)
+
+    override fun popularMangaRequest(page: Int): Request = GET("$baseUrl/drecomics/series", headers)
+
+    override fun popularMangaParse(response: Response): MangasPage {
+        val document = response.asJsoup()
+        val mangas = document.select(".seriesList__item").map {
+            SManga.create().apply {
+                setUrlWithoutDomain(it.selectFirst("a")!!.attr("href"))
+                title = it.selectFirst(".seriesList__text")!!.text()
+                thumbnail_url = it.selectFirst("img")?.attr("src")
+            }
+        }
+        return MangasPage(mangas, false)
+    }
+
+    override fun searchMangaRequest(page: Int, query: String, filters: FilterList): Request {
+        val filter = filters.firstInstance<TypeFilter>()
+        val url = baseUrl.toHttpUrl().newBuilder()
+            .addPathSegments(filter.toUriPart().removePrefix("/"))
+            .build()
+        return GET(url, headers)
+    }
+
+    override fun searchMangaParse(response: Response): MangasPage {
+        if (response.request.url.toString().contains("/drestudios")) {
+            val document = response.asJsoup()
+            val mangas = document.select(".seriesLineupStudios_item").map {
+                SManga.create().apply {
+                    setUrlWithoutDomain(it.attr("href"))
+                    title = it.selectFirst(".seriesLineupStudios_itemTitle")!!.text()
+                    thumbnail_url = it.selectFirst("img")?.attr("src")
+                }
+            }
+            return MangasPage(mangas, false)
+        }
+        return popularMangaParse(response)
+    }
+
+    override fun mangaDetailsParse(response: Response): SManga {
+        val document = response.asJsoup()
+        val isWebtoon = response.request.url.toString().contains("/drestudios")
+
+        return SManga.create().apply {
+            if (isWebtoon) {
+                title = document.selectFirst(".detailStudios_title span")!!.text()
+                author = document.select(".detailStudios_author p").eachText().joinToString(", ")
+                description = document.selectFirst(".detailStudios_storySynopsis")?.text()
+                genre = document.select(".seriesDetail__genreLink--studios").eachText().joinToString(", ")
+                thumbnail_url = document.selectFirst(".detailStudios_mainLeft img.img-fluid")?.attr("src")
+            } else {
+                title = document.selectFirst(".detailComics_title span")!!.text()
+                author = document.select(".detailComics_authorsItem").eachText().joinToString(", ")
+                description = document.selectFirst(".detailComics_synopsis")?.text()
+                genre = document.select(".detailComics_genreListItem").eachText().joinToString(", ")
+                thumbnail_url = document.selectFirst(".detailComicsSection .img-fluid")?.attr("src")
+            }
+        }
```

Comment:
`, ` is the default separator and can be omitted.
```suggestion
        return SManga.create().apply {
            if (isWebtoon) {
                title = document.selectFirst(".detailStudios_title span")!!.text()
                author = document.select(".detailStudios_author p").eachText().joinToString()
                description = document.selectFirst(".detailStudios_storySynopsis")?.text()
                genre = document.select(".seriesDetail__genreLink--studios").eachText().joinToString()
                thumbnail_url = document.selectFirst(".detailStudios_mainLeft img.img-fluid")?.attr("src")
            } else {
                title = document.selectFirst(".detailComics_title span")!!.text()
                author = document.select(".detailComics_authorsItem").eachText().joinToString()
                description = document.selectFirst(".detailComics_synopsis")?.text()
                genre = document.select(".detailComics_genreListItem").eachText().joinToString()
                thumbnail_url = document.selectFirst(".detailComicsSection .img-fluid")?.attr("src")
            }
        }
```